### DISCUSSION TITLE
Use of QGC and Dronekit

### DISCUSSION THREAD

**Discussion by Sasysan** [RosettaDrone/rosettadrone#134] [Q&A]
I try to use QGC and Dronekit ant the same time with the app ( i enable also the second telemetry) but I noticed that it is not possible to send two mavlink messages even if secondary telemetry is enabled. To send and receive mavlink messages only the first telemetry works while the second only allows reception

**Comment by kripper** [comment]
Hi,

Just yesterday I committed an implementation of listeners on every telemetry socket.

I also added a third telemetry socket in the settings.
Unlimited sockets can be handled now and they are all symmetric (send an and receive).

And don't ask me how I knew you would be asking for this today when nobody did for many years :-)
